crate/test_server/benches/benches.rs

```diff
@@ -48,7 +48,9 @@ fn bench_rsa_create_keypair(c: &mut Criterion) {
             .get_or_try_init(start_default_test_kms_server)
             .await
             .unwrap();
-        ctx.owner_client_conf.initialize_kms_client().unwrap()
+        ctx.owner_client_conf
+            .initialize_kms_client(None, None)
+            .unwrap()
```

Comment:
panic risk.

Reply:
thanks for notifying it. As discussed, unwrap() are allowed only in tests and benches.